src/components/EpigraphyView.vue

```diff
@@ -1,20 +1,29 @@
 <template>
   <div>
-    <h1 class="text-3xl text-gray-700">
-      Epigraphies
-    </h1>
-    
-    <div v-for="(lines, side) in data" :key="side">
-      <h3 class="text-2xl text-gray-700 mt-3">
-        <span class="text-gray-700">
+    <v-progress-circular
+      v-if="loading"
```

Comment:
I think this is like a progress spinner animation that only shows if loading == true?

Reply:
That's correct.